t a/sql-src/mariadb-10.4.6/storage/mroonga/ha_mroonga.cpp b/sql-src/mariadb-10.4.6/storage/mroonga/ha_mroonga.cpp
index a2ab9dd..e6e7f85 100644
--- a/sql-src/mariadb-10.4.6/storage/mroonga/ha_mroonga.cpp
+++ b/sql-src/mariadb-10.4.6/storage/mroonga/ha_mroonga.cpp
@@ -553,6 +553,12 @@ static const char *mrn_inspect_extra_function(enum ha_extra_function operation)
   case HA_EXTRA_FAKE_START_STMT:
     inspected = "HA_EXTRA_FAKE_START_STMT";
     break;
+  case HA_EXTRA_NEXT_VALUE:
+    inspected = "HA_EXTRA_NEXT_VALUE";
+    break;
+  case HA_EXTRA_SET_VALUE:
+    inspected = "HA_EXTRA_SET_VALUE";
+    break;
 #ifdef MRN_HAVE_HA_EXTRA_EXPORT
   case HA_EXTRA_EXPORT:
     inspected = "HA_EXTRA_EXPORT";
